packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx

```diff
@@ -78,6 +90,7 @@ export const CallContent = ({
   ParticipantsInfoBadge,
   VideoRenderer,
   layout = 'grid',
+  reactions,
```

Comment:
I would recommend renaming it to `supportedReactions`.

Reply:
Done ✅